clay/clay-kiln
# Custom sort

### Description

* New feature that enables sorting pages by field inside the pages drawer
* Modifies `page-list-header` UI
* Adds sorting logic
* This is based off issue #1403 

### QA

**NOTE:** In order to test this feature, we need to `npm link` the new changes from `amphora-search` in branch `update-page-mapping`

* Link this package to a clay instance with `npm link`
* Create a couple of articles with different titles an authors
* Verify that sorting is working correctly for `title`, `byline` and `status`
* Collaborators field shouldn't be clickable
